### DISCUSSION TITLE
The training wait a long time after load model.

### DISCUSSION THREAD

**Discussion by daiwc** [STVIR/pysot#186] [General]
[2019-09-23 11:06:50,663-rk0-train.py#311] model prepare done
[2019-09-23 11:06:50,664-rk0-train.py#174] model
.backbone (AlexNet)
         .layer1 (Sequential)
                .0 (Conv2d)
                  - weight
                  - bias
                .1 (BatchNorm2d)
                  - weight
                  - bias
                .2 (MaxPool2d)
                .3 (ReLU)
         .layer2 (Sequential)
                .0 (Conv2d)
                  - weight
                  - bias
                .1 (BatchNorm2d)
                  - weight
                  - bias
                .2 (MaxPool2d)
                .3 (ReLU)
         .layer3 (Sequential)
                .0 (Conv2d)
                  - weight
                  - bias
                .1 (BatchNorm2d)
                  - weight
                  - bias
                .2 (ReLU)
         .layer4 (Sequential)
                .0 (Conv2d)
                  - weight
                  - bias
                .1 (BatchNorm2d)
                  - weight
                  - bias
                .2 (ReLU)
         .layer5 (Sequential)
                .0 (Conv2d)
                  - weight
                  - bias
                .1 (BatchNorm2d)
                  - weight
                  - bias
.rpn_head (DepthwiseRPN)
         .cls (DepthwiseXCorr)
             .conv_kernel (Sequential)
                         .0 (Conv2d)
                           - weight
                         .1 (BatchNorm2d)
                           - weight
                           - bias
                         .2 (ReLU)
             .conv_search (Sequential)
                         .0 (Conv2d)
                           - weight
                         .1 (BatchNorm2d)
                           - weight
                           - bias
                         .2 (ReLU)
             .head (Sequential)
                  .0 (Conv2d)
                    - weight
                  .1 (BatchNorm2d)
                    - weight
                    - bias
                  .2 (ReLU)
                  .3 (Conv2d)
                    - weight
                    - bias
         .loc (DepthwiseXCorr)
             .conv_kernel (Sequential)
                         .0 (Conv2d)
                           - weight
                         .1 (BatchNorm2d)
                           - weight
                           - bias
                         .2 (ReLU)
             .conv_search (Sequential)
                         .0 (Conv2d)
                           - weight
                         .1 (BatchNorm2d)
                           - weight
                           - bias
                         .2 (ReLU)
             .head (Sequential)
                  .0 (Conv2d)
                    - weight
                  .1 (BatchNorm2d)
                    - weight
                    - bias
                  .2 (ReLU)
                  .3 (Conv2d)
                    - weight
                    - bias

the training don't go on after load model, even though i wait  many hours.
the command that i used  is  CUDA_VISIBLE_DEVICES=0,1 python -m torch.distributed.launch --nproc_per_node=2 --master_port=2333  ../../tools/train.py --cfg config.yaml

**Comment by chun-chun123** [comment]
I have met the same question.Have you managed it?

**Comment by ZhiyuanChen** [comment]
What is your batch size?

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing it for now.
Please reopen it or create a new issue if you have further question.
